test/unit/plugins/provisioners/ansible/provisioner_test.rb

```diff
@@ -583,6 +583,33 @@ def self.it_should_create_and_use_generated_inventory(with_ssh_user = true)
       end
     end
 
+    describe "with galaxy support" do
+
+      before do
+        config.galaxy_role_file = existing_file
+      end
+
+      it "raises an error when ansible-galaxy command fails", skip_before: true, skip_after: true do
+        config.finalize!
+        Vagrant::Util::Subprocess.stub(execute: Vagrant::Util::Subprocess::Result.new(1, "", ""))
+
+        expect {subject.provision}.to raise_error(VagrantPlugins::Ansible::Errors::AnsibleCommandFailed)
+      end
+
+      it "execute ansible-galaxy and ansible-playbook" do
+        # TODO: to be improved, but I'm currenty facing some issues, maybe only present in RSpec 2.14...
```

Comment:
I get:

```
     Failure/Error: expect(Vagrant::Util::Subprocess).to receive(:execute).with("ansible-galaxy", *args).ordered
     NameError:
       undefined local variable or method `args' for #<RSpec::Core::ExampleGroup::Nested_1::Nested_1::Nested_17:0x007fe1e1aa5850>
```

I also tried things like
```
expect(Vagrant::Util::Subprocess).to receive(:execute).with("ansible-galaxy", anything).ordered
expect(Vagrant::Util::Subprocess).to receive(:execute).with("ansible-playbook", anything).ordered
```
but also without success:

```
     Failure/Error: expect(Vagrant::Util::Subprocess).to receive(:execute).with("ansible-galaxy", anything).ordered
       <Vagrant::Util::Subprocess (class)> received :execute with unexpected arguments
         expected: ("ansible-galaxy", #<RSpec::Mocks::ArgumentMatchers::AnyArgMatcher:0x007f899c124940>)
              got: ("ansible-galaxy", "install", "--role-file=/tank/gildegoma/wip/github/mitchellh/vagrant/test/unit/plugins/provisioners/ansible/provisioner_test.rb", "--roles-path=/var/folders/.../vagrant20151120-55224-19ecpf3/work/roles", "--force", {:notify=>[:stdout, :stderr], :workdir=>"/var/folders/.../vagrant20151120-55224-19ecpf3/work"}), ("ansible-playbook", "--connection=ssh", "--timeout=30", "--limit=machine1", "--inventory-file=/var/folders/.../vagrant20151120-55224-19ecpf3/work/.vagrant_resi/provisioners/ansible/inventory", "playbook.yml", {:env=>{"PYTHONUNBUFFERED"=>1, "ANSIBLE_FORCE_COLOR"=>"true", "ANSIBLE_HOST_KEY_CHECKING"=>"false", "ANSIBLE_SSH_ARGS"=>"-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o ControlMaster=auto -o ControlPersist=60s"}, :notify=>[:stdout, :stderr], :workdir=>"/var/folders/.../vagrant20151120-55224-19ecpf3/work"})
```
Maybe worth mention in the last error, that both command arguments (for galaxy and playbook) are received. I'm confused... (sleep time here)